webservices/tasks/download.py

```diff
@@ -168,7 +168,7 @@ def make_manifest(resource, row_count, path):
         fp.write('Time: {}\n'.format(resource['timestamp']))
         fp.write('Resource: {}\n'.format(resource['path']))
         fp.write('*Count: {}\n'.format(row_count))
-        fp.write('Filters:\n\n')
+        fp.write('Fields:\n\n')
```

Comment:
I was in the process of adding them, I wanted that to be a separate pr, I thought I rolled that back.